src/SDK/Trace/SpanExporter/SpanExporterFactoryInterface.php

```diff
@@ -9,4 +9,6 @@
 interface SpanExporterFactoryInterface
 {
     public function create(): SpanExporterInterface;
+    public function type(): string;
+    public function priority(): int;
```

Comment:
As above with an [example](https://github.com/opentelemetry-php/exporter-zipkin/blob/main/_register.php#L5) from the Zipkin exporter.